app/src/main/java/org/mozilla/fenix/compose/SelectableChip.kt

```diff
@@ -32,12 +33,16 @@ import org.mozilla.fenix.theme.Theme
  *
  * @param text [String] displayed in this chip. Ideally should only be one word.
  * @param isSelected Whether this should be shown as selected.
+ * @param selectedBackgroundColor Optional background color when the chip is selected.
+ * @param unselectedBackgroundColor Optional background color when the chip is not selected.
```

Comment:
```suggestion
 * @param selectedBackgroundColor Optional background [Color] when the chip is selected.
 * @param unselectedBackgroundColor Optional background [Color] when the chip is not selected.
```
Don't know what you think of this? Might be nice to just reference to the type when it's easy to do so.

Reply:
I think it's a good enhancement